packages/integration-tests/utils/helpers.ts

```diff
@@ -146,6 +146,23 @@ async function getMultipleSentryTransactionRequests(page: Page, count: number, u
   return getMultipleRequests(page, count, envelopeUrlRegex, envelopeRequestParser, url);
 }
 
+/**
+ * Get current Sentry session at the given URL, or the current page
+ *
+ * @param {Page} page
+ * @param {string} [url]
+ * @return {*}  {Promise<Session>}
+ */
+async function getCurrentSession(page: Page, url?: string): Promise<Session> {
+  if (url) {
+    await page.goto(url);
+  }
+
+  const session = await page.evaluateHandle<Session>('window.Sentry.getCurrentHub().getScope().getSession()');
+
+  return session.jsonValue();
```

Comment:
I would prefer if we assert on the session envelopes instead of grabbing it this way.